kamelets/jms-ibm-mq-sink.kamelet.yaml

```diff
@@ -0,0 +1,124 @@
+    camel.apache.org/kamelet.support.level: "Preview"
+    camel.apache.org/provider: "Apache Software Foundation"
+    camel.apache.org/catalog.version: "main-SNAPSHOT"
+    camel.apache.org/kamelet.group: "JMS"
+  labels:
+    camel.apache.org/kamelet.type: "sink"
+    camel.apache.org/requires.runtime: camel-k
+spec:
+  definition:
+    title: "JMS - IBM MQ Kamelet Sink"
+    description: "A Kamelet that can produce events to an IBM MQ message
+    queue using JMS."
+    required:
+      - host
+      - port
+      - destinationName
+      - queueManager
+    type: object
+    properties:
+      host:
+        title: "Broker host"
+        description: "The JMS host"
+        type: string
+        example: "localhost"
+      port:
+        title: "Broker port"
+        description: "The JMS port"
+        type: integer
+        example: 1414
+      destinationType:
+        title: "Destination Type"
+        description: "The JMS destination type (i.e.: queue or topic)"
+        type: string
+        default: queue
+      destinationName:
+        title: "Destination Name"
+        description: "The JMS destination name"
+        type: string
+        example: DEV.APP.SVRCONN
+      queueManager:
+        title: "Queue Manager"
+        description: "Name of the Queue Manager"
+        type: string
+        example: QM1
+      username:
+        title: "Username"
+        description: "Username to authenticate, if needed"
+        type: string
+        example: app
+      password:
+        title: "Password"
+        description: "Password to authenticate, if needed"
+        type: string
+  dependencies:
+  - "camel:jms"
+  - "camel:kamelet"
+  - "mvn:com.ibm.mq:com.ibm.mq.allclient:9.2.5.0"
```

Comment:
This needs to be added as description, we cannot add this dependency explicitly in the release. It's not compliant with ASF policy.